exchanges/coinut/coinut.go

```diff
@@ -197,16 +197,16 @@ func (c *COINUT) GetTradeHistory(ctx context.Context, instrumentID, start, limit
 // GetIndexTicker returns the index ticker for an asset
 func (c *COINUT) GetIndexTicker(ctx context.Context, asset string) (IndexTicker, error) {
 	var result IndexTicker
-	params := make(map[string]interface{})
+	params := make(map[string]any)
 	params["asset"] = asset
 
 	return result, c.SendHTTPRequest(ctx, exchange.RestSpot, coinutIndexTicker, params, false, &result)
 }
 
 // GetDerivativeInstruments returns a list of derivative instruments
-func (c *COINUT) GetDerivativeInstruments(ctx context.Context, secType string) (interface{}, error) {
-	var result interface{} // to-do
-	params := make(map[string]interface{})
+func (c *COINUT) GetDerivativeInstruments(ctx context.Context, secType string) (any, error) {
+	var result any // to-do
```

Comment:
🐒 Unusual spelling of TODO won't get highlighted (at least for me)
```suggestion
	var result any // TODO: Provide a concrete type for the result
```